build.rs

```diff
@@ -1,6 +1,18 @@
 // (c) Copyright 2019-2024 OLX
 fn main() {
-    println!("cargo:rustc-link-lib=vips");
-    println!("cargo:rustc-link-lib=glib-2.0");
-    println!("cargo:rustc-link-lib=gobject-2.0");
+    if cfg!(target_os = "windows") {
+        //
+        // Get the binaries for Windows from the link below.
+        // https://github.com/libvips/build-win64-mxe/releases/
+        //
+        // Use Windows binaries with the suffix `-ffi.zip` .
+        //
+        println!("cargo:rustc-link-lib=libvips");
+        println!("cargo:rustc-link-lib=libglib-2.0");
+        println!("cargo:rustc-link-lib=libgobject-2.0");
+    } else {
```

Comment:
maybe prefer early return instead of an else?

Reply:
nit btw

Reply:
Sure, why not